src/types.ts

```diff
@@ -150,26 +152,23 @@ export interface BinaryRecordedEvent<Type extends string = string>
    * Data of this event.
    */
   data: Uint8Array;
-
-  /**
-   * Representing the metadata associated with this event.
-   */
-  metadata: Uint8Array;
 }
 
 export interface AllStreamJSONRecordedEvent<
   Type extends string = string,
   Data = unknown,
-  Metadata extends Record<string, unknown> = Record<string, unknown>
+  Metadata = unknown
```

Comment:
**QUESTION:** Can we make this type more specific?

Reply:
We could, it its probably not too useful. JSON.parse can return object, array, string, number, null and then we also have Uint8Array. At this point, you will need either narrow the type yourself, (what `unknown` forces you to do) or you cast the type. I think `unknown` adds more clarity.

Reply:
I personally to be more explicit where it's possible, but I'm fine with keeping it unknown.